src/Packetery/Module/Options/Page.php

```diff
@@ -480,22 +480,22 @@ public function create_form(): Form {
 			->toggle( '#packetery-default-dimensions-value' );
 
 		$container->addText( 'default_length', __( 'Length', 'packeta' ) . ' (mm)' )
-			->addRule( Form::INTEGER )
-			->addRule( Form::MIN, null, 0 )
 			->addConditionOn( $form[ self::FORM_FIELDS_CONTAINER ]['default_dimensions_enabled'], Form::EQUAL, true )
-			->setRequired();
+			->setRequired()
+			->addRule( Form::INTEGER )
+			->addRule( Form::MIN, null, 0 );
 
 		$container->addText( 'default_height', __( 'Height', 'packeta' ) . ' (mm)' )
-			->addRule( Form::INTEGER )
-			->addRule( Form::MIN, null, 0 )
 			->addConditionOn( $form[ self::FORM_FIELDS_CONTAINER ]['default_dimensions_enabled'], Form::EQUAL, true )
-			->setRequired();
+			->setRequired()
+			->addRule( Form::INTEGER )
+			->addRule( Form::MIN, null, 0 );
 
 		$container->addText( 'default_width', __( 'Width', 'packeta' ) . ' (mm)' )
-			->addRule( Form::INTEGER )
-			->addRule( Form::MIN, null, 0 )
 			->addConditionOn( $form[ self::FORM_FIELDS_CONTAINER ]['default_dimensions_enabled'], Form::EQUAL, true )
-			->setRequired();
+			->setRequired()
```

Comment:
odsadil bych o jeden tab všechno pod tou podmínkou

Bude z toho aspoň jasnější, že ty pravidla a setRequired se validuje až po splnění podmínky.

Dívám se, že to tak je i na řádku 470 nebo 476.

Reply:
Ok, udělal jsem tedy jednotné odsazení.